src/main/java/unicash/model/commons/Amount.java

```diff
@@ -57,8 +60,13 @@ public static String amountToDecimalString(Amount amt) {
         return formattedNumberString;
     }
 
+
+    public String originalString() {
+        return Double.toString(amount);
+    }
+
```

Comment:
Is this method still used anywhere, if not maybe we could remove it

Reply:
It's for backwards compatibility with some failing unit tests, one of which requires some toString to be parsed, but I can go back and check if I can change those tests

Reply:
`parseCommand_addTransaction` in UniCashParserTest seems to require it, I think it's supp to parse simulate user inputs, and since the dollar symbol prefixed is not a recognized user input for amount, the regular `toString` method doesn't work. Thinking about, maybe we can allow the user to input a currency symbol as well? like `amt/$40.20` seems like a natural input right